docs/run/run-jobs-batch.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Run jobs in a batch
+description: How to run jobs in batch mode. 
+
+---
+
+# Run jobs in a batch
+
+Batch mode can shorten processing time if all jobs can be provided at the outset. If you want to submit iterative jobs, use [sessions](sessions) instead.  Using batch mode has these benefits:
```

Comment:
Is it possible to add links to new tutorials that use batch mode? 

Later today, we'll be updating the [CHSH](https://learning.quantum-computing.ibm.com/tutorial/chsh-inequality) and [Grover](https://learning.quantum-computing.ibm.com/tutorial/grovers-algorithm) one to do exactly that

Reply:
Of course!

Reply:
done